frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeReplicaSection.tsx

```diff
@@ -21,7 +24,16 @@ const ServingRuntimeReplicaSection: React.FC<ServingRuntimeReplicaSectionProps>
 
   return (
     <FormSection title="Model server replicas">
-      <FormGroup label="Number of model server replicas to deploy">
+      <FormGroup
+        label="Number of model server replicas to deploy"
+        labelIcon={
+          infoContent ? (
+            <Popover bodyContent={<div>{infoContent}</div>}>
+              <HelpIcon aria-label="More info" />
+            </Popover>
```

Comment:
We have a mix of `HelpIcon` and `QuestionCircleIcon` in various parts of the UI. @vconzola which is correct.

Reply:
@uidoyen please change all instances of `HelpIcon` to `QuestionCircleIcon`